ajax/libs/jodit/package.json

```diff
@@ -19,7 +19,7 @@
   "npmName": "jodit",
   "npmFileMap": [
     {
-      "basePath": "",
+      "basePath": "build",
       "files": [
         "*jodit*.+(css|js|map)"
```

Comment:
This would also match the unminified files that are published, these should be included in the PR please :)

Reply:
Added